docs/_docs/live-update/index.md

```diff
@@ -71,3 +71,16 @@ App][3].
 [1]: {%link _docs/live-update/setup.md %}
 [2]: {%link _docs/live-update/design-switcher.md %}
 [3]: {%link _docs/tutorial/index.md %}
+
+## Using an HTTP Proxy Server on Android {#HttpProxy}
+
+Live Update can be configured to download Figma design via an 
+HTTP Proxy Server on Android.
+
+To set the HTTP Proxy Server, run:
+
+```adb shell settings put global http_proxy "<host>:<port>"```
```

Comment:
Should set the code style to use and have the block quotes on different lines.
```suggestion
```shell
adb shell settings put global http_proxy "<host>:<port>"
```
```

Reply:
Done.